### PR TITLE
VetTec: Content Updates 17055, 17015

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17015
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17055

## Testing done
- local testing

## Screenshots
screen shot 2019-02-22 at 2 19 31 pm
screen shot 2019-02-22 at 2 32 58 pm
screen shot 2019-02-22 at 2 32 52 pm

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by devin-mccurdy — APPROVED** [review]
LGTM

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/edu-benefits/0994/containers/IntroductionPage.jsx [hunk 1]
```diff
@@ -69,16 +70,23 @@ export class IntroductionPage extends React.Component {
...
                   After submitting the form, you’ll get a confirmation message.
                   You can print this for your records.
                 </li>
+                <li>
+                  Then, complete an Application for VA Education Benefits (VA
+                  Form 22-1990), if you haven’t already.
+                </li>
+                <li>
+                  <strong>Note:</strong>
+                  If you're already receiving VA education benefits, you’ve
```
Comment (jbalboni) [inline src/applications/edu-benefits/0994/containers/IntroductionPage.jsx]: The `you're` in this line has a quote mark instead of an apostrophe

**Review by jbalboni — APPROVED** [review]
One nit, otherwise looks good.

**Pull request merged**